docs/reference/modules/node.asciidoc

```diff
@@ -22,39 +22,36 @@ dedicated data nodes, {ml} nodes, and {transform} nodes.
 
 <<master-node,Master-eligible node>>::
 
-A node that has `node.master` set to `true` (default), which makes it eligible
-to be <<modules-discovery,elected as the _master_ node>>, which controls the
-cluster.
+A node that has the `master` role (default), which makes it eligible to be
+<<modules-discovery,elected as the _master_ node>>, which controls the cluster.
 
 <<data-node,Data node>>::
 
-A node that has `node.data` set to `true` (default). Data nodes hold data and
-perform data related operations such as CRUD, search, and aggregations.
+A node that has the `data` role (default). Data nodes hold data and perform data
+related operations such as CRUD, search, and aggregations.
 
 <<ingest,Ingest node>>::
 
-A node that has `node.ingest` set to `true` (default). Ingest nodes are able
-to apply an <<pipeline,ingest pipeline>> to a document in order to transform
-and enrich the document before indexing. With a heavy ingest load, it makes
-sense to use dedicated ingest nodes and to mark the master and data nodes as
-`node.ingest: false`.
+A node that has the `ingest` role (default). Ingest nodes are able to apply an
+<<pipeline,ingest pipeline>> to a document in order to transform and enrich the
+document before indexing. With a heavy ingest load, it makes sense to use
+dedicated ingest nodes and to not have the `ingest` role on master` and `data`.
```

Comment:
Typo:

```suggestion
dedicated ingest nodes and to not have the `ingest` role on `master` and `data`.
```

I'm not sure about the wording. Maybe:


```suggestion
dedicated ingest nodes and to remove the `ingest` role from nodes that have the `master` or `data` roles.
```

Reply:
Thanks I incorporated both of these.